Cargo.toml

```diff
@@ -15,16 +15,16 @@ edition = "2021"
 
 [dependencies]
 csv = "1.2"
-tui = "0.19"
-crossterm = { version = "0.26.1", features = ["use-dev-tty"] }
+ratatui = "0.25.0"
+crossterm = { version = "0.27.0", features = ["use-dev-tty"] }
 anyhow = "1.0"
 clap = { version = "4.2", features = ["derive"] }
 tempfile = "3.5"
 regex = "1.8"
 csv-sniffer = "0.3.1"
 
 [target.'cfg(windows)'.dependencies]
-crossterm = "0.25"
+crossterm = "0.27.0"
```

Comment:
Is it possible keep this pinned at 0.25 for windows? This is due to an on-going issue in crossterm. More info: https://github.com/YS-L/csvlens/issues/30

Reply:
Fixed in https://github.com/YS-L/csvlens/pull/42/commits/3ce170a62be29f47121a85496bff5a9f6e543112

Usually apps can filter to only include `key_event.kind == Press` to workaround this.
Is there a need to handle release events in this app? I didn't see anything obvious.

See https://github.com/ratatui-org/ratatui/issues/347 for more info.

Reply:
Nice! I think this should work. Thanks for the fix.